Xamarin.Essentials/SecureStorage/SecureStorage.shared.cs

```diff
@@ -5,7 +5,7 @@ namespace Xamarin.Essentials
 {
     public static partial class SecureStorage
     {
-        internal static readonly string Alias = Preferences.PrivatePreferencesSharedName;
+        internal static readonly string Alias = Preferences.GetPrivatePreferencesSharedName("securestorage");
```

Comment:
This is going to break existing users.  I know we aren't stable yet, but there may be some folks using this in production and I think we should special case `"securestorage"` to return the previous storage name.